config/locales/notices/en.yml

```
@@ -15,6 +15,11 @@ en:
       use_diff_email:
         link: create a new account
         text_html: Or, %{link} using a different email address.
+    maintenance:
+      contact_us: Contact us
+      currently_under_maintenance_html: We are currently <b>under maintence until
```

Typo:
```suggestion
      currently_under_maintenance_html: We are currently <b>under maintenance until
```

Also, I could see an argument that `<b>` is fine to use, though any sight of the element raises a red flag for me to consider [`strong`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong) instead.